docs/making-your-own.md

```diff
@@ -135,31 +123,44 @@ see [Local Testing](/local-testing)
-    - [Kinoite](https://docs.fedoraproject.org/en-US/fedora-kinoite/)
     - [Silverblue](https://docs.fedoraproject.org/en-US/fedora-silverblue/)
-- Additionally, your developer users might not be familiar with [Podman](https://podman.io/)
+    - [Kinoite](https://docs.fedoraproject.org/en-US/fedora-kinoite/)
+    - [CoreOS](https://docs.fedoraproject.org/en-US/fedora-coreos/)
+    - [startingpoint](https://github.com/ublue-os/startingpoint)
+- You and your users might be more used to traditional desktops
+    - On immutable systems, most software development tools shouldn't be installed directly on your base system. They should be run separate from the host system in some way, like using [a toolbox](https://universal-blue.org/guide/toolbox/) or [Podman](https://podman.io/) directly.
     - Most documentation around "How do I install a webserver in Fedora" will not apply
     - It looks more like `podman run --name podman-nginx -p 8080:80 -d nginx`
-    - IDEs are still a problem area 
 
 ### Set up the pipeline first
 
 Don't try to add something complicated out of the box, start with something basic, make a small change. 
 Then ensure that the builds are working, are signed properly, and you get used to making changes in your repository.
-This will save you a ton of time, because you can rev faster long term instead of getting stuck on something unrelated, spending a ton of time fixing it, and then realizing that you still have to fix the rest of the pipe.
+This will save you a ton of time, because you can do revisions faster long term instead of getting stuck on something unrelated, spending a ton of time fixing it, and then realizing that you still have to fix the rest of the pipe.
 
-- Get the water flowing first!
+> Get the water flowing first!
 
 ### Resist the urge to add the entire universe 
- - Systems like this are designed for a _small, lean, mean, maintainable, and performant core_. Remember that updates to the _base image_ require a reboot, so ideally you want that surface area to be small - let Flatpak handle the rest.
+ - Systems like this are designed for a _small, lean, mean, maintainable, and performant core_. Remember that updates to the _base image_ require a reboot, so ideally you want that surface area to be small - let Flatpak and other userspace tools handle the rest.
  - But also remember that these systems are atomic, you don't need to manually clean up an old decision, the user will just get a new pristine image (ideally every day), so if you need to add a bunch of packages to get the desired outcome then you can always trim it down later - just remember that you need to account for the user's data!
- - This means: be cognizant where you put stuff. When making an image you can't put things in `/usr/local` or other common places to put custom things, in this context you're the distro now: so probably `/usr/bin`. Don't worry, you got this.
-
 
+### Remember where you _can_ add files
```

Comment:
We should also mention that /usr can't be modified after the ostree is committed though I think that should go in a general users guide

Reply:
> after the ostree is committed

Do you mean that it can't be modified after boot or after `ostree container commit`.

Reply:
It applies in either case. I think mentioning after boot keeps it simple